src/library/libraryview.h

```diff
@@ -38,8 +38,7 @@ class LibraryView {
     }
     /// If applicable, requests that the LibraryView changes the track color of
     /// the selected track. Does nothing otherwise.
-    virtual void assignPreviousTrackColor() {
-    }
-    virtual void assignNextTrackColor() {
+    virtual void selectTrackColor(int steps) {
+        Q_UNUSED(steps)
     }
```

Comment:
https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Rf-unused
```suggestion
    virtual void selectTrackColor(int) {
    }
```